chain/client/src/client.rs

```diff
@@ -1506,122 +1506,134 @@ impl Client {
         }
 
         if let Some(validator_signer) = self.validator_signer.clone() {
```

Comment:
This was refactored to two new methods:
- reconcile_transaction_pool
- produce_chunks